src/pages/CropSidebar/CropSidebar.js

```diff
@@ -173,9 +166,9 @@ const CropSidebar = ({
       return true;
     });
     dispatchRedux(updateActiveCropData(filtered.map((filter) => filter.id)));
-  }, [sfilters.cropSearch, cropDataRedux, dispatchRedux, sfilters]);
+  }, [filters.cropSearch, cropDataRedux, dispatchRedux, filters]);
```

Comment:
having filters.cropsearch and filters in the dependency array is redundant. you can just use filters

Reply:
ohhh good catch! i didnt see that.